src/test/ui/uninhabited/uninhabited-matches-feature-gated.rs

```diff
@@ -20,10 +20,10 @@ fn main() {
     let _ = match x {}; //~ ERROR non-exhaustive
 
     let x: (Void,) = unsafe { std::mem::uninitialized() };
-    let _ = match x {}; //~ ERROR non-exhaustive
+    let _ = match x {}; // okay
 
     let x: [Void; 1] = unsafe { std::mem::uninitialized() };
-    let _ = match x {}; //~ ERROR non-exhaustive
+    let _ = match x {}; // okay
```

Comment:
Why do these change?

Reply:
Because pattern matching now uses the more precise uninhabited checks: https://github.com/rust-lang/rust/pull/54125/files/7e14ffb3622848d8cd481482f943dec235afb797#diff-a2cadcdf8ac2f01fa5a02f2b4e0a8267R232.

Reply:
D'oh I must have been tired. I thought this was a "unreachable code" warning being code, but really it's just the compiler finally understanding that this match is exhaustive. Nice!